packages/request-dashboard/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@requestnetwork/create-request-form",
+  "version": "0.1.1",
+  "main": "./dist/web-component.umd.cjs",
+  "scripts": {
+    "dev": "vite dev",
+    "build": "vite build && npm run package && npm run build:wc",
+    "build:wc": "vite build -c vite.wc.config.ts",
+    "preview": "vite preview",
+    "package": "svelte-kit sync && svelte-package && npm run build:wc && publint",
+    "prepublishOnly": "npm run package",
+    "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+    "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
+    "clean": "rm -rf dist && rm -rf .svelte-kit",
+    "is-release-needed": "bash ../../scripts/is-release-needed.sh"
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "svelte": "./dist/index.js",
+      "require": "./dist/web-component.umd.cjs",
+      "default": "./dist/web-component.js"
+    }
+  },
+  "files": [
+    "dist",
+    "!dist/**/*.test.*",
+    "!dist/**/*.spec.*"
+  ],
+  "peerDependencies": {
+    "svelte": "^4.0.0"
```

Comment:
`svelte` is listed as a peerDependency and as a devDependency. Please de-duplicate.